### PR TITLE
Fixes for cloning - unset props on clone.

### PR DESCRIPTION
**Pull request by turbo124**

### CODE REVIEW

File: src/pages/invoices/edit/components/Actions.tsx [hunk 1]
```diff
@@ -101,7 +140,15 @@ export function useActions() {
       ...(invoice as unknown as RecurringInvoice),
       number: '',
       documents: [],
-      frequency_id: '5'
+      frequency_id: '5',
```
Comment (beganovich) [inline src/pages/invoices/edit/components/Actions.tsx:143]: This should *probably* be part of enums.

File: src/pages/invoices/edit/components/Actions.tsx [hunk 2]
```diff
@@ -112,7 +159,15 @@ export function useActions() {
...
       number: '',
       documents: [],
-      date: dayjs().format('YYYY-MM-DD')
+      date: dayjs().format('YYYY-MM-DD'),
+      total_taxes: 0,
+      exchange_rate: 1,
+      last_sent_date: '',
+      project_id: '',
+      subscription_id: '',
+      status_id: '1',
```
Comment (beganovich) [inline src/pages/invoices/edit/components/Actions.tsx:168]: This also.

File: src/pages/recurring-invoices/common/hooks.tsx [hunk 3]
```diff
@@ -344,7 +366,15 @@ export function useActions() {
...
       number: '',
       documents: [],
-      date: dayjs().format('YYYY-MM-DD')
+      date: dayjs().format('YYYY-MM-DD'),
+      total_taxes: 0,
+      exchange_rate: 1,
+      last_sent_date: '',
+      project_id: '',
+      subscription_id: '',
+      status_id: '1',
```
Comment (beganovich) [inline src/pages/recurring-invoices/common/hooks.tsx:375]: This also.

**Review by beganovich — APPROVED** [review]
Looks good, I wonder if we can improve it and secure it more with enums.

**Comment by turbo124** [comment]
We can iterate on this, can we merge for now and add another task to jira to refactor with enums? Just to ensure functionality operates correctly.

**Comment by beganovich** [comment]
Sure, works for me.

**Pull request merged**